source/utils/backoff.ts

```diff
@@ -0,0 +1,5 @@
+export default async function backoff(ms: number): Promise<void> {
+	await new Promise(resolve => {
+		setTimeout(resolve, ms);
+	});
+}
```

Comment:
```suggestion
export default async function delay(milliseconds: number): Promise<void> {
	await new Promise(resolve => {
		setTimeout(resolve, milliseconds);
	});
}
```

The file should be renamed too.

Reply:
Yep, done, thanks!